libcst/metadata/tests/test_scope_provider.py

```diff
@@ -1609,6 +1610,27 @@ def test_no_out_of_order_references_in_global_scope(self) -> None:
                     ),
                 )
 
+    def test_walrus_accesses(self) -> None:
+        if sys.version_info < (3, 8):
```

Comment:
Same thing here?

Reply:
This one is not as easy, I'd have to plumb the parser config through some helper functions, so I thought it's fine to keep it as is